docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1221,17 +1213,158 @@ paths:
             $ref: '#/definitions/Problem'
         '409':
           description: |
-            Conflict. There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            Conflict. There are several possible reasond for receiving this status code:
+
+            - There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            already equals the maximal value - the total amount of this subscription partitions.
+
+            - Resetting of subscription cursors request is still in the progress.
+
+            - This endpoint was called for a subscription with balance_mode set to 'custom'.
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Access is forbidden for the client or event type
+          schema:
+            $ref: '#/definitions/Problem'
+
+    post:
+      tags:
+        - subscription-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        GET with body.
+        Starts a new stream for reading events from this subscription. Using this endpoint it is possible to specify
+        which partitions of subscription to read in this stream.
+
+        It is possible to specify partition that is already consumed by another stream, in this case there are two
+        possible outcomes. If the partition was assigned to another stream as a result of automatic assignment - it will
+        be possible to "steal" it. But if the partition is assigned to another stream because it was directly requested
+        by it - then it will not be possible to transfer this partition to current stream - such request will fail.
```

Comment:
It is possible to specify a partition that is already consumed by another stream. In this case, there can be two outcomes, depending on how the current assignment was obtained. (1) If the partition is already assigned to another stream through automatic assignment, it will be possible to "steal" it, and the request will succeed. (2) If the partition is already assigned to another stream because of a request to stream from this partition specifically, then it will not be possible to "steal" the partition, and the request will fail.

Reply:
you also haven't defined "steal" :-P

Reply:
Yes, I saw @SergKam 's comment after I wrote this :)